### PR TITLE
Update 1-a-Python.ipynb

### PR DESCRIPTION
**Pull request by quicktus**
fixed some small typos/ inconsistencies/ ambiguities.

### CODE REVIEW

File: exercise/1-Python-Pandas/1-a-Python.ipynb [hunk 1]
```diff
@@ -68,20 +68,20 @@
    "source": [
     "## Basic Data Types\n",
     "\n",
-    "- Numbers: integer, float, complex\n",
-    "- String `\"I'm a string\"`\n",
+    "- Number: integer, float, complex\n",
```
Comment (melsigl) [inline exercise/1-Python-Pandas/1-a-Python.ipynb:71]: Keep "Numbers" as you can have these three distinct types of numbers. Alternatively, you may call them "Numerical" if you want to stick with singular naming.

File: exercise/1-Python-Pandas/1-a-Python.ipynb [hunk 2]
```diff
@@ -68,20 +68,20 @@
...
-    "- Dictionaries: `{key1: value1, key2: value2}`\n"
+    "- Dictionary: `{key1: value1, key2: value2}`\n"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Numbers"
+    "### Number"
```
Comment (melsigl) [inline exercise/1-Python-Pandas/1-a-Python.ipynb:84]: Same here: Keep "Numbers" as you can have these three distinct types of numbers. Alternatively, you may call them "Numerical" if you want to stick with singular naming.

**Comment by melsigl** [comment]
Thank you for your PR. In the future, please only create a PR from your feature branch so that we do not clutter our commit messages with PRs from forks.

**Comment by quicktus** [comment]
As suggested, "Numbers" has been changed to "Numerical" to maintain a consistent singular nomenclature.

**Pull request merged**